### DISCUSSION TITLE
I have been trying to swap my token

### DISCUSSION THREAD

**Discussion by Joe4b2k** [aptos-labs/aptos-developer-discussions#551] [General]
I have been trying to swap my coins from cellana back to aptos but it was showing me was 
Max gas units below min transaction on gas units 
I even try sending out coin same thing

**Comment by Pcharlesme** [comment]
@Joe4b2k 
Recommended Solutions:

Increase max_gas_amount: Ensure that the max_gas_amount in your transaction is above the minimum required. The Aptos blockchain has a parameter txn.min_transaction_gas_units that defines this minimum. Make sure your max_gas_amount exceeds this value. 

Adjust Gas Unit Price: Verify that your gas_unit_price meets or exceeds the network's minimum. Transactions with a gas unit price below the minimum (txn.min_price_per_gas_unit) will be rejected. 

Check Account Balance: Ensure your account has sufficient funds to cover the maximum possible gas fee. If your account balance is lower than max_gas_amount * gas_unit_price, the transaction will fail.

**Reply by jkjun4574** [reply]
> [!TIP]

**Reply by jkjun4574** [reply]
> @Joe4b2k 추천 솔루션:
> 
> max_gas_amount 증가: 거래의 max_gas_amount가 필요한 최소값보다 높은지 확인하십시오. Aptos 블록체인에는 이 최소값을 정의하는 매개변수 txn.min_transaction_gas_units가 있습니다. max_gas_amount가 이 값을 초과하는지 확인하십시오.
> 
> 가스 단위 가격 조정: 귀하의 가스_단위_가격이 네트워크의 최소값을 충족하거나 초과하는지 확인하십시오. 가스 단위 가격이 최소값(txn.min_price_per_gas_unit) 미만인 거래는 거부됩니다.
> 
> 계좌 잔액 확인: 계좌에 가능한 최대 가스 요금을 충당할 수 있는 충분한 자금이 있는지 확인하세요. 계좌 잔액이 max_gas_amount * gas_unit_price보다 낮으면 거래가 실패합니다.

max_gas_amount
